prisma/schema.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE "User" (
+    "id" TEXT NOT NULL,
```

Comment:
Should `id` be type `TEXT`? What do you think of type `SERIAL`?

Reply:
sounds good - done.